knowledge_repo/app/auth_providers/oauth2.py

```diff
@@ -32,9 +27,9 @@
         'scopes': None,
         'user_info_endpoint': 'user',
         'user_info_mapping': {
-            'identifier': 'email',
+            'identifier': 'login',
```

Comment:
I have a preference that the identifier always be emails where possible, since it makes sharing of knowledge posts between instances a sane experience, and allows for collaborations between different institutions.

Reply:
Updated to email, login as users don't have to have an email for github. Specifically, my email comes back as None (even though github certainly has it?)